config/routes.rb

```diff
@@ -37,6 +37,7 @@
   end
   resources :sensor_readings, default: { format: :json }
   post 'sensor_readings/fake', to: 'sensor_readings#fake', default: { format: :json }
+  post 'sensor_readings/add', to: 'sensor_readings#add'
```

Comment:
obsolete, if you use `resources :sensor_readings` see the Rails guides: http://guides.rubyonrails.org/routing.html